src/thunderbots/software/network_input/filter/robot_filter.h

```diff
@@ -19,7 +19,7 @@ typedef struct SSLRobotData_t
     Angle orientation;
     double confidence;
     Timestamp timestamp;
-} SSLRobotData;
+} SSLRobotDetection;
```

Comment:
Does this need to be public?

Reply:
Yes it should be public, `SSLRobotDetections` are constructed in the backend before being passed to the filter. This abstracts the filter from needing to know about the protobuf packet structure/data.